Rubberduck.Core/UI/Command/ComCommands/ComCommandBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using NLog;
+using Rubberduck.VBEditor.Events;
+
+namespace Rubberduck.UI.Command.ComCommands
+{
+    public abstract class ComCommandBase : CommandBase
+    {
+        private readonly IVBEEvents _vbeEvents;
+
+        protected ComCommandBase(ILogger logger, IVBEEvents vbeEvents) : base(logger)
```

Comment:
Probably out of scope, but wouldn't it be simpler to just grab the class logger in the ctor instead of having it in every single derived class's `base` call? I'm guessing this was initially done because `GetCurrentClassLogger()` returns the abstract base class logger (I consider that a bug, [NLog disagrees](https://github.com/NLog/NLog/issues/2141)), but there's [a work-around here](https://stackoverflow.com/a/35119353/4088852).

Reply:
This is an pre-existing thing, and I think it's best that this is addressed in its own PR - I'm sure it's not just the commands that does this -- we have lot of inconsistencies in how we initialize the logger all over the place, I think.